doc/source/cluster/kubernetes/user-guides/tpu.md

```diff
@@ -0,0 +1,71 @@
+(kuberay-tpu)=
+
+# Using TPUs with KubeRay
+
+This document provides tips on TPU usage with KubeRay.
+
+TPUs are available on Google Kubernetes Engine (GKE). To use TPUs with Kubernetes, configure
+both your Kubernetes setup and add additional values to your RayCluster configuration.
+Instructions for configuring a RayCluster with TPUs on GKE can be found here:
+
+- {ref}`kuberay-gke-tpu-cluster-setup`
+
+## Quickstart: Serve a Stable Diffusion model on GKE with TPUs
+
+After setting up a GKE cluster with TPUs and the Ray TPU initialization webhook, you're ready to begin running
+workloads on Ray with TPUs. The :ref:`StableDiffusion example <kuberay-tpu-stable-diffusion-example>` shows how to
+serve a model with Ray on single-host TPUs.
+
+
+## Configuring Ray Pods for TPU usage
+
+Using any TPU accelerator requires specifying `google.com/tpu` resource `limits` and `requests` in the container fields of your `RayCluster`'s
+`workerGroupSpecs`. This resource specifies the number of TPU chips for GKE to allocate each Pod. KubeRay v1.1.0 adds a `numOfHosts`
+field to the RayCluster custom resource, specifying the number of TPU hosts to create per worker group replica. For multi-host worker groups,
+replicas are treated as PodSlices rather than individual workers, with `numOfHosts` worker nodes being created per replica.
+Additionally, GKE uses `gke-tpu` node selectors to schedule TPU pods on the node matching the desired TPU accelerator and topology.
+
+Below is a config snippet for a RayCluster worker group with 2 Ray TPU workers, each scheduled on their own GKE v4 TPU node.
+
+```
+   groupName: tpu-group
```

Comment:
Do we need to set `rayStartParams`?

Reply:
I don't think so, the resource gets detected from the k8s container limits so setting `google.com/tpu: "4"` should be sufficient

Reply:
how does Ray schedule the Ray tasks/actors to a TPU node if we don't specify `rayStartParams`?

Reply:
I'm thinking of the code [here](https://github.com/ryanaoleary/ray/blob/9a2f2c262a46f167eb742533966c66cc75b95743/python/ray/autoscaler/_private/kuberay/autoscaling_config.py#L269) that adds the "TPU" resource if it's detected in the container limits but not set in `rayStartParams`. I'm not sure if that's just used for autoscaling, but I tested with this example [RayService ](https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray-cluster.tpu-v4-singlehost.yaml) that doesn't set `rayStartParams`, and the [Ray actor](https://github.com/ray-project/serve_config_examples/blob/d16752eef753c3eb8efa8bb37d0ba0891bc875cd/stable_diffusion/stable_diffusion_tpu.py#L34) that requests `"resources": {"TPU": 4}` was scheduled correctly.